projects/user_publication/utils.py

```diff
@@ -42,7 +53,11 @@ def guess_project_for_publication(authors, pub_year):
     in the publication's list of authors.
     """
     from projects.models import Project, ProjectPIAlias
-
+    try:
+        pub_year = int(pub_year)
```

Comment:
Under what circumstances would this happen? I think if the caller passes an invalid year, I would consider that an invariant, and the code should not continue executing.

Are there any valid scenarios where a publication could have a non-integer year?